test/test.safe-eval.js

```diff
@@ -1,77 +1,98 @@
-import {checkBuiltInVMAndNodeVM} from '../test-helpers/checkVM.js';
+import {checkBuiltInVMAndNodeVM} from "../test-helpers/checkVM.js";
```

Comment:
Is this passing linting? The linting should normally be enforcing single quotes...

Reply:
Yes, it is passing lint and was done by autoformat only.
I also noticed that some files use `"` ( https://github.com/JSONPath-Plus/JSONPath/blob/39ee1844c610f9fd8925d3095ed3971e3d48025e/test/test.cli.js#L1 ) while some use `'` ( https://github.com/JSONPath-Plus/JSONPath/blob/39ee1844c610f9fd8925d3095ed3971e3d48025e/test/test.eval.js#L1 ), suggesting that linting/formatting tooling is broken or not strictly defined.

I tried checking out to very old commits (from 2022) and using different editors (neovim, vscode), it still formats the same.

Reply:
I've removed the unnecessary formatting changes in this file. We can handle format/lint issue separately.
Also updated the docs & badges.